app.js

```diff
@@ -1,44 +1,102 @@
-var express = require('express');
-var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
+// Get dependencies
+const express = require('express');
+const path = require('path');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const http = require('http');
+const passport = require('passport');
+const flash = require('connect-flash');
+const session = require('express-session');
+const async = require('async');
+const nodemailer = require('nodemailer');
+const bcrypt = require('bcrypt-nodejs');
```

Comment:
please only have code that is currently used. any future plans should be explained in comments instead. This way, we can ensure there isn't unnecessary code at any given time.